roles/v2/mining-proxy/src/lib/upstream_mining.rs

```diff
@@ -497,9 +709,44 @@ impl
 
     fn handle_open_extended_mining_channel_success(
         &mut self,
-        _m: OpenExtendedMiningChannelSuccess,
+        m: OpenExtendedMiningChannelSuccess,
     ) -> Result<SendTo<DownstreamMiningNode>, Error> {
-        todo!("450")
+        let extranonce_prefix: Extranonce = m.extranonce_prefix.clone().try_into().unwrap();
+        let range_0 = 0..m.extranonce_prefix.clone().to_vec().len();
+        let range_1 = (range_0.end)..EXTRANOUNCE_RAGE_1_LENGTH;
```

Comment:
EXTRANONCE_RANGE_1_LENGTH Is the correct spelling.

Reply:
thanks